Tools/Scripts/webkitpy/common/system/executive.py

```diff
@@ -391,17 +391,20 @@ def run_command(self,
                              pass_fds=pass_fds)
         with process:
             if not string_to_communicate:
-                output = process.communicate()[0]
+                output = process.communicate(timeout=Timeout.difference())[0]
```

Comment:
Do we not want to catch the `TimeoutException` here, so that we can handle that case (and, e.g., kill the process, rather than leaving it alive)?